integrations/apps/mailgun/views.py

```diff
@@ -498,3 +498,43 @@ class IideaContacView(MailgunGenericContactView):
     EMAIL_TEMPLATE = 'email/generic_contact.html'
     FROM_TEXT = 'Iidea'
     SUBJECT = 'Nuevo contacto desde página web:Iidea'
+
+
+class AlianzaContactView(MailgunGenericContactView):
+    KEY = settings.MAILGUN_API_KEY
+    DOMAIN = settings.RER_MAILGUN_DOMAIN
+    RECIPIENT = settings.RER_MAILGUN_RECIPIENT
+    EMAIL_TEMPLATE = 'email/alianza_contact.html'
+    FROM_TEXT = 'Alianza SweMex'
+    SUBJECT = 'Nuevo contacto desde página web'
+
+    @method_decorator(csrf_exempt)
```

Comment:
The `generic_contact` content all variables to need

https://github.com/BlickLabs/integrations/blob/985aca37ca06f0f2e5cd27f3192722f5e93c082e/integrations/templates/email/generic_contact.html

See https://github.com/BlickLabs/integrations/blob/985aca37ca06f0f2e5cd27f3192722f5e93c082e/integrations/apps/mailgun/views.py#L14 for example of use generic contact